configs/foreman/2.5.yaml

```
@@ -0,0 +1,541 @@
+---
+:project: foreman
+:github_org: theforeman
+:releases:
+  :2.5.0:
+    :redmine_version_id: 1319
```

```suggestion
    :redmine_version_id: 1367
```

(not sure it's used anywhere, but yeah)

---

Thanks!